incuna-sass/helpers/_pseudo-elements.sass

```diff
@@ -1,13 +1,10 @@
 // Documentation https://gist.github.com/3339249
```

Comment:
all these files still need to be documented, even the legacy stuff, so that will happen